Formula/graph-tool.rb

```diff
@@ -111,7 +114,12 @@ def install
       package = Formula[package_name].opt_libexec
       (libexec/site_packages/"homebrew-#{package_name}.pth").write package/site_packages
     end
-
+    
+    ENV["CC"]=Formula["llvm"].opt_bin/"clang"
+    ENV["CXX"]=Formula["llvm"].opt_bin/"clang++"
+    ENV["LDFLAGS"]="-L#{Formula["llvm"].opt_lib} -Wl,-rpath,#{Formula["llvm"].opt_lib} -L#{Formula["libomp"].opt_lib} -Wl,-rpath,#{Formula["libomp"].opt_lib}"
+    ENV["CPPFLAGS"]="-I#{Formula["llvm"].opt_include} -I#{Formula["libomp"].opt_include}"
```

Comment:
That's way too much custom patching. Why not just use the macOS Accelerate implementation?

Reply:
OpenMP is not supported by the default clang in MacOS. That's why we need to use llvm and libomp. AFAIK, there's no other way to enable it than what is done above.

Reply:
Yes, so why not use the macOS alternative to OpenMP?